configs/projects/pe-bolt-server.rb

```diff
@@ -15,30 +33,17 @@
                  "/var/run/puppetlabs/#{service}")
   end
 
-  proj.setting(:prefix, "/opt/puppetlabs/server/apps/bolt-server")
   proj.setting(:pe_bolt_user, "pe-bolt-server")
-  proj.setting(:bindir, "#{proj.prefix}/bin")
-  proj.setting(:libdir, "#{proj.prefix}/lib")
   proj.setting(:homedir, "/opt/puppetlabs/server/data/bolt-server")
-  proj.setting(:gem_home, File.join(proj.libdir, 'ruby'))
-  proj.setting(:gem_install, "/opt/puppetlabs/puppet/bin/gem install --no-rdoc --no-ri --local --bindir=#{proj.bindir}")
   proj.setting(:gem_build, "/opt/puppetlabs/puppet/bin/gem build")
-  proj.setting(:artifactory_url, "https://artifactory.delivery.puppetlabs.net/artifactory")
-  proj.setting(:buildsources_url, "#{proj.artifactory_url}/generic/buildsources")
 
   proj.user(proj.pe_bolt_user,
             group: proj.pe_bolt_user,
             shell: '/sbin/nologin',
             homedir: "#{proj.homedir}",
             is_system: true)
 
-  # R10k dependency
-  proj.component 'rubygem-gettext-setup'
-  # Bolt dependencies that aren't included in Puppet on our platforms yet
-  proj.component 'rubygem-ffi'
-  proj.component 'rubygem-minitar'
-
-  proj.instance_eval File.read('configs/projects/bolt-shared.rb')
+  proj.component 'bolt-runtime'
```

Comment:
I feel like i'm missing something. Should this just instead be pulling in pe-bolt-server-runtime? https://github.com/puppetlabs/puppet-runtime/pull/166/commits/5eb70d408f6ddb42f98c658c3200c550becbff5a#diff-ce229c5167a3d6e25ac9eccf8fb4ad08

Reply:
oh yeah, I should have left an explanatory comment -- I've left the 'bolt-runtime' component in this repo as-is, since it reads the name of the puppet-runtime project to download from the settings -- [here's](https://github.com/puppetlabs/bolt-vanagon/pull/99/files#diff-92ba05fa40b223f0030f004fd11b1a3bR15) where the pe-bolt-server name is set. The only job of the bolt-runtime component in this repo is pretty much to just build the tarball URL, download it, and untar it.

Reply:
Once pe-bolt-server is moved to a separate repo, this will make more sense again

Reply:
Awesome. Thanks for schooling me.